packages/features/users/components/UserTable/EditSheet/EditUserSheet.tsx

```diff
@@ -0,0 +1,249 @@
+                <div className="mt-4 flex items-center gap-2">
+                  <Avatar
+                    asChild
+                    className="h-[36px] w-[36px]"
+                    alt={`${loadedUser?.name} avatar`}
+                    imageSrc={avatarURL}
+                    gravatarFallbackMd5="fallback"
+                  />
+                  <div className="space-between flex flex-col leading-none">
+                    <Skeleton loading={isLoading} as="p" waitForTranslation={false}>
+                      <span className="text-emphasis text-lg font-semibold">
+                        {loadedUser?.name ?? "Nameless User"}
+                      </span>
+                    </Skeleton>
+                    <Skeleton loading={isLoading} as="p" waitForTranslation={false}>
+                      <p className="subtle text-sm font-normal">
+                        {orgBranding?.fullDomain ?? WEBAPP_URL}/{loadedUser?.username}
+                      </p>
+                    </Skeleton>
+                  </div>
+                </div>
+                <div className="mt-6 flex flex-col space-y-5">
+                  <DisplayInfo label={t("email")} value={loadedUser?.email ?? ""} displayCopy />
+                  <DisplayInfo
+                    label={t("bio")}
+                    badgeColor="gray"
+                    value={loadedUser?.bio ? loadedUser?.bio : t("user_has_no_bio")}
+                  />
+                  <DisplayInfo label={t("role")} value={loadedUser?.role ?? ""} asBadge badgeColor="blue" />
+                  <DisplayInfo label={t("timezone")} value={loadedUser?.timeZone ?? ""} />
+                  <DisplayInfo
+                    label={t("availability_schedules")}
+                    value={
+                      schedulesNames && schedulesNames?.length === 0
+                        ? [t("user_has_no_schedules")]
+                        : schedulesNames ?? "" // TS wtf
+                    }
+                  />
+                  <DisplayInfo
+                    label={t("teams")}
+                    displayCount={teamNames?.length ?? 0}
+                    value={
+                      teamNames && teamNames?.length === 0 ? [t("user_isnt_in_any_teams")] : teamNames ?? "" // TS wtf
+                    }
+                    asBadge={teamNames && teamNames?.length > 0}
+                  />
+                </div>
+              </div>
+            ) : (
+              <div className="flex-grow">
+                <EditForm
+                  selectedUser={loadedUser}
+                  avatarUrl={avatarURL}
+                  domainUrl={orgBranding?.fullDomain ?? WEBAPP_URL}
+                  dispatch={dispatch}
+                />
+              </div>
+            )}
+            <SheetFooter className="mt-auto">
+              <SheetFooterControls />
+            </SheetFooter>
+          </div>
+        ) : (
+          <Loader />
+        )}
+      </SheetContent>
+    </Sheet>
+  );
+}
```

Comment:
Quite a big file, not sure if it makes sense to separate in different files

Reply:
Will do this in the morning